server/database/data/label.json

```diff
@@ -0,0 +1,40 @@
+[
+    {
+        
+        "name": "Végétarien",
+        "image": " ../../../client/public/images/Vegetarien.png",

```

Comment:
the right syntax is : 
"image": " /images/Vegetarien.png"

Reply:
Ok syntax changed